subworkflows/local/variant_calling/call_snv_sentieon.nf

```diff
@@ -24,7 +27,6 @@ workflow CALL_SNV_SENTIEON {
 		ch_versions = Channel.empty()
 
         SENTIEON_DNASCOPE ( input, fasta, fai, dbsnp, dbsnp_index, call_interval, ml_model )
-        ch_vcf      = SENTIEON_DNASCOPE.out.vcf
```

Comment:
ch_vcf : we have to keep this!  vcf from dnascope is sent to following functions when there is no ml file. 
and also the same channel for the output of bcftools_two. One of them is sent to the join function afterwards.